### PR TITLE
Use `Resolves` in PR template to link issue to new PR

### LINKED ISSUE #241: Change PR template to auto-link it to the issue in description
In GitHub, there's a feature that automatically links an issue to a PR if the issue is mentioned in the description using a keyword (such as `closes` and more).
See here for more info: https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

Currently, the PR template is:
```
Connected to #[relevant GitHub issues, eg 76]
```
This is **not** one of the keywords, so we do not benefit from this feature.

To use this feature, `Connected to` should be replaced with one of the following:
1. Closes
1. Fixes
1. Resolves

I think `Resolves` best matches, so it should read:
```
Resolves #[relevant GitHub issues, eg 76]
```

### PR DESCRIPTION
**Pull request by abrahamko**
In GitHub PR template, replace issue placeholder to make use of GitHub feature
that automatically links issue to PR.
For that, the issue should be prefixed by a 'resolved' keyword.

### What does this PR do?
Replace `Connected to` with `Resolves` when prefixing the related issue in PR template.
This enables GitHub auto-linking feature: https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

### What ticket does this PR close?
Resolves #241

#### Change log
- [ ] The CHANGELOG has been updated, or
- [x] This PR does not include user-facing changes and doesn't require a CHANGELOG update

#### Test coverage
- [ ] This PR includes new unit and integration tests to go with the code changes, or
- [x] The changes in this PR do not require tests

#### Documentation
- [ ] Docs (e.g. `README`s) were updated in this PR, and/or there is a follow-on issue to update docs, or
- [x] This PR does not require updating any documentation

### CODE REVIEW

**Review by orenbm — APPROVED** [review]
Please wait for @izgeri 's approval to verify there isn't a reason we used `Connected to`.

**Review by izgeri — APPROVED** [review]
LGTM. I also changed this in `cyberark/conjur-template` (so new repos will use this syntax) and filed cyberark/community#80 to update all repos with this (great!) change

**Pull request merged**
